README.md

```diff
@@ -19,6 +19,10 @@ html-duration-picker.js is a very tiny JS library used for transforming a native
 
 ![Duration Picker Input](https://github.com/nadchif/html-duration-picker.js/blob/master/screenshots/default-after.jpg)
 
+**Hide Seconds** ```<input type="text" class="html-duration-picker" data-hide-seconds>```
```

Comment:
Please move this to [DOCUMENTATION.md](https://github.com/nadchif/html-duration-picker.js/blob/master/DOCUMENTATION.md)